src/main/kotlin/racingCar/domain/Race.kt

```diff
@@ -0,0 +1,19 @@
+package racingCar.domain
+
+import kotlin.random.Random
+
+private const val RANDOM_NUM_RANGE = 10
+private const val REFERENCE_VAL = 4
+fun racingRule(): Boolean {
+    return Random.nextInt(RANDOM_NUM_RANGE) >= REFERENCE_VAL
```

Comment:
```suggestion
    return Random.nextInt(0..9) >= REFERENCE_VAL
```

다른 곳에서 활용해주신 것처럼, range 를 활용해주면 보다 더 직관적인 표현이 가능하겠네요!